table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -201,6 +209,11 @@ public SyncStatus sync(JournalContext context) throws IOException {
     // Synchronization is necessary if accessed concurrently from multiple threads
     SyncStatus.Builder builder = SyncStatus.newBuilder();
 
+    if (Files.exists(Paths.get(mConfigPath))) {
+      ObjectMapper mapper = new ObjectMapper();
+      DbConfig dbConfig = mapper.readValue(new File(mConfigPath), DbConfig.class);
+      mBypassSet.addAll(dbConfig.getBypassSet());
+    }
```

Comment:
I think if we follow the format of configuration as:
`{  "bypassSet" : ["table1", "table2"]}`, it will have problem in some cases.
For example, we have db1:{table1, table2, table3}, db2:{table1, table2, table3}. If we only want to bypass the table1 in db1, but we will also bypass table1 in db2?

Reply:
The configuration is related each database instance. So, for db1, we use a configuration file db1.config.json, for db2 is db2.config.json.